.github/workflows/release-arm.yaml

```diff
@@ -81,9 +81,9 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         run: |
           curl https://luet.io/install.sh | sudo sh
-          IMG=$(cat build/IMAGE).img
+          IMAGE=$(cat build/IMAGE).img
```

Comment:
the problem of using `build/IMAGE` is that inside it already contains something along the lines of `FLAVOR:TAG` however, for the img files, the repository includes `img` as a suffix. So we will need to be able to say here something like `FLAVOR-img:TAG`

Reply:
correct, we moved the .img images in a different repo, so this won't make use of them.. however we could merge this and look at that again later, wdyt?

Reply:
I think in that case, having the other solution would have been a better in between step cuz at least we would release the img files and then we can fix the issue for good with however we want the script to look like